src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -15,11 +15,11 @@
 
 # pylint: disable=g-statement-before-imports
 try:
-  # ClusterFuzz dependencies.
-  from python.base import modules
-  modules.fix_module_search_paths()
+    # ClusterFuzz dependencies.
```

Comment:
nit: I don't think we want to change the indentation here or in the rest of the file right?